.github/workflows/clang-tidy.yml

```diff
@@ -0,0 +1,23 @@
+name: Clang Tidy
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Install clang-tidy
+      run: |
+        sudo apt-get update -qyy
+        sudo apt-get install clang-tidy-8 -qyy
```

Comment:
Doesn't github have a way to pre-install packages without doing it manually?

Reply:
i'm searching a way but it doesn't seems, looked for this project while searching, but github is less flexible  than gitlab which permits to use any docker image
https://github.com/onqtam/doctest/pull/285/files

Reply:
but i take the point that i should convert everything in a single MR :)

Reply:
it seems no, it's as raw as gitlab